bucket/multipar-beta.json

```diff
@@ -9,15 +9,15 @@
         "par1j.exe",
         "par2j.exe",
         "par2j64.exe",
-        "sfv_md5.exe",
-        "tool\\par2_rename.exe"
+        "sfv_md5.exe"
     ],
     "shortcuts": [
         [
             "MultiPar.exe",
             "MultiPar"
         ]
     ],
+    "notes": "Some Python script files are provided in .\\tool folder.",
```

Comment:
Could you put notes after license? The conventional order is here fyi: https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets

Could you also replace `.\\tool` with `$dir\\tool` - it will be expanded to the full installation dir for the user

Reply:
> Could you put notes after license? The conventional order is here fyi: https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets
> 
> Could you also replace `.\\tool` with `$dir\\tool` - it will be expanded to the full installation dir for the user

Done. And also put `pre_install` in the right place according to guidelines.